diagnostic.md

```diff
@@ -33,7 +38,11 @@ y = 10
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
 <!-- solution below -->
-
+Line:
+1- x=4
+2- y=3
+3- x=3
+4- y=10
```

Comment:
These are right but you didn't answer the final two questions.
- The variables don't change, they remain `x` and `y`.
- The values do change as you assign them.